src/OpenLoco/EditorController.cpp

```diff
@@ -82,17 +84,17 @@ namespace OpenLoco::EditorController
             Game::sub_46DB4C(); // draw preview map
         }
 
-        options.maxCompetingCompanies = gameState.maxCompetingCompanies;
-        options.competitorStartDelay = gameState.competitorStartDelay;
+        options.maxCompetingCompanies = CompanyManager::getMaxCompetingCompanies();
+        options.competitorStartDelay = CompanyManager::getCompetitorStartDelay();
         gameState.numberOfIndustries = options.numberOfIndustries;
-        options.objectiveType = gameState.objectiveType;
-        options.objectiveFlags = gameState.objectiveFlags;
-        options.objectiveCompanyValue = gameState.objectiveCompanyValue;
-        options.objectiveMonthlyVehicleProfit = gameState.objectiveMonthlyVehicleProfit;
-        options.objectivePerformanceIndex = gameState.objectivePerformanceIndex;
-        options.objectiveDeliveredCargoType = gameState.objectiveDeliveredCargoType;
-        options.objectiveDeliveredCargoAmount = gameState.objectiveDeliveredCargoAmount;
-        options.objectiveTimeLimitYears = gameState.objectiveTimeLimitYears;
+        options.objectiveType = enumValue(Scenario::getObjective().type);
+        options.objectiveFlags = Scenario::getObjective().flags;
```

Comment:
Do the options just contain a full instance of the structure

Reply:
The first 8 variables are identical, but Options version is missing the last 3. Could always move these last 3 into another substruct called ObjectiveTime or something, and then Options can have Objective, and GameState can have Objective+ObjectiveTime or similar. Maybe in another PR?

Reply:
Yes that sounds like a good idea. I'll approve this for now